src/main/kotlin/de/nycode/bankobot/variables/parsers/calc/CalcExpression.kt

```diff
@@ -37,24 +37,32 @@ class CalcExpression(val input: String) : Expression<BigDecimal> {
     private var result: BigDecimal? = null
 
     override fun getResult(): BigDecimal {
-        if (result != null) {
-            return result as BigDecimal
-        }
+        try {
+            if (result != null) {
+                return result as BigDecimal
+            }
 
-        val input = CharStreams.fromString(input)
-        val lexer = CalculationLexer(input).apply {
-            removeErrorListeners()
-            addErrorListener(ThrowingErrorListener)
-        }
+            val input = CharStreams.fromString(input)
```

Comment:
Is there a reason for this ident?

Reply:
wtf it's not so in my ide
![image](https://user-images.githubusercontent.com/37078297/106459431-22fde700-6492-11eb-8b8f-2520423aaf38.png)